package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
`index.js` file should be started from this scripts segment, how do you do that using nodemon?

Reply:
You are right, however, the task is asking you:
- `index.js file is started via npm script command using nodemon`

so the idea is executing the script using a command like: `npm start` or `npm run get-random-numer`, if we try to run the npm script you created using `npm start`, it will run the file using `node` instead of `nodemon`, so you need to change it to make it work as it is requested.

Almost there!

Reply:
In that capture, you are not using a npm script or command, you are using nodemon directly. The idea of task's request is to execute the script but using a npm command, for example: try running: `npm start` in the terminal, and check if nodemon is being used or it is using node.

For more information: https://docs.npmjs.com/cli/v10/using-npm/scripts

Reply:
Oh, I FINALLY get it

But at the same time, the task wasn't very clear, like, at all, neither were the clarifications :/

Reply:
Okay, I finally get it

I'm sorry, but both the clarifications and the task itself were very confusing and it wasn't very clear what was expected at all.

Reply:
No problem, I am going to be more clear the next time. Once you fix it, you can submit again the task in the learn portal to set your final grade.

Good job!